app/MasterTemplate.hs

```diff
@@ -11,6 +11,8 @@ import Config (enableFb, enableCdn)
 import Util.Blaze
 import Scripts (globalScripts)
 
+
```

Comment:
@frankjohnlee please delete these two blank lines (and in general review your commits so that you don't include extra whitespace).